src/components/elements/Map-mapbox/utils.ts

```diff
@@ -176,6 +189,72 @@ export const addGeojsonToDraw = (geojson: any, uuid: string, cb: Function, curre
   }
 };
 
+export const addMediaSourceAndLayer = (map: mapboxgl.Map, modelFilesData: GetV2MODELUUIDFilesResponse["data"]) => {
+  const layerName = "media-images";
+  removeMediaLayer(map);
+  removePopups("MEDIA");
+  const modelFilesGeolocalized = modelFilesData!.filter(
+    model => model.location && model.location.lat && model.location.lng
+  );
+  if (modelFilesGeolocalized.length === 0) {
+    return;
+  }
+
+  const features: Feature<Geometry, GeoJsonProperties>[] = modelFilesGeolocalized.map(modelFile => ({
+    type: "Feature",
+    geometry: {
+      type: "Point",
+      coordinates: [modelFile.location?.lng, modelFile.location?.lat]
```

Comment:
These are already filtered to only include model files that are guaranteed to have a valid location, so these `?.` are unnecessary. If they're here because of TS compiler constraints, something like this should work

```
coordinates: [modelFile.location!.lng, modelFile.location.lat]

```

Reply:
done